projects/front-end-library/src/lib/components/input-range/scss/index.scss

```
@@ -0,0 +1,228 @@
+/* *****************************************************************************
+*
+*  Input Range
+*
+* ***************************************************************************** */
+
+.bf-input-range {
+    max-width: 1200px;
+    width: 100%;
+    padding: 0$spacer-2;
+
+    /*----------------------------------------------*/
+    /* Overlay of the input range double slider
+    /*----------------------------------------------*/
+    &_panel {
+        position: relative;
+        height:  1px;
+        text-align: left;
+        margin: 2rem 0 0;
+        width: 100%;
+    }
+    &_overlay {
+        position: absolute;
+        left: 0;
+        right: 0;
+        height:  1px;
+        margin: 0$spacer-2;
+    }
+    &_exclude {
+        position: absolute;
+        height:  1px;
+        top: -1px;
+        background-color: rgba(var(--bf-color-neutral-primary-rgb), .35) ;
+
+        &--left {
+            left: 0;
+            &:before {
+                position: absolute;
+                left: -$spacer-2;
+                width:$spacer-2;
+                height: 1px;
+                background-color: inherit;
+                content: ' ';
+            }
+        }
+        &--right {
+            right: 0;
+            &:after {
+                position: absolute;
+                right: -$spacer-2;
+                width:$spacer-2;
+                height: 1px;
+                background-color: inherit;
+                content: ' ';
+            }
+        }
+    }
+    &_value {
+        position: absolute;
+        left: 0;
+        top: -2px;
+        height: 3px;
+        margin-inline-start: 0;
+        background-color: var(--bf-color-neutral-primary);
+        z-index: 1;
+        &:before {
+            position: absolute;
+            left: -$spacer-2;
+            width:$spacer-2;
+            height: 3px;
+            background-color: inherit;
+            content: ' ';
+        }
+        &:after {
+            position: absolute;
+            right: -$spacer-2;
+            width:$spacer-2;
+            height: 3px;
+            background-color: inherit;
+            content: ' ';
+        }
+
+    }
+    &_handle {
+        position: absolute;
+        width: 0;
+        height: 0;
+        background-color: transparent;
+        top: -2rem;
+        text-align: center;
+        z-index: 25;
+    }
+    &_label {
+        position: absolute;
+        display: block;
+        width: 3rem;
+        left: -1.5rem;
+        z-index:3;
+        color: var(--bf-color-neutral-primary);
+    }
+    /*----------------------------------------------*/
+    /* component's custom elements
+    /*----------------------------------------------*/
+    &_legend{
```

```suggestion
    &_legend {
```
Fait attention au espace. Prends le temps de regardé tes fichiers

---

Corrige partout. Je ne peux pas te les pointer